packages/core-plugin-v1/src/proxies/service-proxy.ts

```diff
@@ -0,0 +1,543 @@
+/**
+ * Service Proxy - Maps V1 services to V2 services
+ *
+ * This file contains proxy implementations for V1 services that delegate to V2 services.
+ * It allows V1 plugins to interact with V2 services through familiar interfaces.
+ */
+
+import { ServiceType as V1ServiceType, Service as V1Service } from '../types';
+import {
+  ServiceType as V2ServiceType,
+  IAgentRuntime as V2IAgentRuntime,
+  Service as V2Service,
+} from '@elizaos/core-plugin-v2';
+import { CompatAgentRuntime } from '../runtime';
+
+/**
+ * Maps a V1 service type to a V2 service type
+ * @param v1Type The V1 service type
+ * @returns The corresponding V2 service type
+ */
+export function mapServiceType(v1Type: V1ServiceType): string {
```

Comment:
perhaps we should change the v2 official type, maybe less enum and more string to allow flexibility